client/src/pages/Login/Login.tsx

```diff
@@ -37,26 +37,25 @@ export default function Login(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={8} md={7} className={classes.container}>
         <Box
           display="flex"
           justifyContent="space-between"
           alignItems="flex-start"
           flexDirection="column"
           className={classes.authWrapper}
         >
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
+                <Typography style={{ fontWeight: 'bold', fontSize: 32 }} className={classes.welcome} variant="h4">
+                  Login
                 </Typography>
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Sign up" />
```

Comment:
```suggestion
            <AuthFooter linkTo="/signup" asideText="Don't have an account?" btnText="Sign up" />
```
Let's rename this component to make more sense :)